.eslintrc.json

```diff
@@ -8,7 +8,8 @@
     },
     "extends": "eslint:recommended",
     "parserOptions": {
-        "sourceType": "module"
+        "sourceType": "module",
+        "ecmaVersion": 2017
```

Comment:
Added for async / await https://github.com/eslint/eslint/issues/8366#issuecomment-326776306